src/isar/config/default.ini

```diff
@@ -131,7 +144,7 @@ vertical_reference_system = MSL
 
 # Rotational representations of reported results
 # Options [quaternion]
-sensor_carrier_orientation_reference_system = quaternion
+media_orientation_reference_system = quaternion
```

Comment:
Is media the right word here? And is quternion a reference system?

Reply:
I struggle to come up with a better name

Reply:
While I agree it's not a perfect name it is the name specified by the data scheme from SLIMM which means it could be more confusing to actually change it :)

Reply:
Aha, then it makes sense to keep it